### ISSUE TITLE
[Feature Request] Callable for some parameters, the same way as LR scheduler

### ISSUE THREAD

**Issue by minhlong94** [DLR-RM/stable-baselines3#540]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.

### 🚀 Feature

Add support for some parameters as a callable function like in this [docs](https://stable-baselines3.readthedocs.io/en/master/guide/examples.html#learning-rate-schedule) example about `learning_rate`. For example: `ent_coef`, `vf_coef`

### Motivation

For algorithms like A2C and PPO, when set `ent_coef = ...`, it is fixed and cannot be changed. Since `learning_rate` can be passed as a callable, these parameters should too. Naively, I want the entropy coef to be high in the beginning, and decays gradually when training.

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by araffin** [comment]
Duplicate of https://github.com/DLR-RM/stable-baselines3/issues/316
